ch01/01_is_unique.go

```diff
@@ -30,3 +32,21 @@ func IsUniqueVanilla(s string) bool {
 
 	return true
 }
+
+// IsUniqueBits determines if a string has all unique chars (only works for ASCII)
+// This solution doesn't utilize any additional data structures and has O(n) time complexity
+func IsUniqueBits(s string) bool {
+
+	s = strings.ToLower(s)
+	var vector int32
+	sBytes := []byte(s) // only for ASCII
+	for _, sByte := range sBytes {
+		index := sByte - 'a'
```

Comment:
```suggestion
	for _, rune := range s {
		index := rune - 'a'
```
In Go [a string is a read-only slice of bytes](https://go.dev/blog/strings#what-is-a-string-h2); we don't need to explicitly convert it to byte slice to pass it to `range`.